### PR TITLE
Zulip: Add version 3.0.0

### PR DESCRIPTION
**Pull request by hoilc**

### CODE REVIEW

File: bucket/zulip.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            "Zulip"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/zulip/zulip-desktop"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/zulip/zulip-desktop/releases/download/v$version/zulip-$version-x64.nsis.7z"
```
Comment (linsui) [inline bucket/zulip.json]:
```suggestion
                "url": "https://github.com/zulip/zulip-desktop/releases/download/v$version/zulip-$version-x64.nsis.7z",
                "hash": {
                    "url": "$baseurl/latest.yml",
                    "regex": "(?sm)x64.*?$base64"
                }
```

File: bucket/zulip.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            },
+            "32bit": {
+                "url": "https://github.com/zulip/zulip-desktop/releases/download/v$version/zulip-$version-ia32.nsis.7z"
```
Comment (linsui) [inline bucket/zulip.json]:
```suggestion
                "url": "https://github.com/zulip/zulip-desktop/releases/download/v$version/zulip-$version-ia32.nsis.7z",
                "hash": {
                    "url": "$baseurl/latest.yml",
                    "regex": "(?sm)ia32.*?$base64"
                }
```

**Comment by Ash258** [comment]
Why not go with universal hash?

```yml
hash:
  url: $baseurl/latest
  regex: (?sm)sha512:\\s+$base64.*?$basename
```

This should work without problems and get proper hashes.

_Sorry for yml format I am on phone an yml is fater to type_

**Comment by linsui** [comment]
It will match the first hash

**Pull request merged**